providers/os/resources/packages/appx_manifest.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package packages
+
+import "encoding/xml"
+
+// AppxManifest represents the structure of an AppxManifest.xml file
+type AppxManifest struct {
```

Comment:
I would like to see a test here where we parse a manifest file in go test.

Reply:
done

Reply:
also added a test for `toPackage` which convers the appxPackage to just a package